Tests/SwerverTests/RequestSpec/HttpRequestParserSpec.swift

```diff
@@ -0,0 +1,23 @@
+import Quick
+import Nimble
+import Swerver
+import Foundation
+
+class HttpRequestParserSpec: QuickSpec {
+    override func spec() {
+        describe("An HTTP Request Parser") {
+            var httpRequestParser: HttpRequestParser!
+            
+            beforeEach {
+                httpRequestParser = HttpRequestParser()
+            }
+            
+            it("returns an HttpRequest Object") {
+                let requestData = "GET / HTTP/1.1"
+                let expectedRequest = HttpRequest(method: RequestMethods.get, url: URL(string: "/")!, httpVersion: HttpVersion.current)
+                let parsedRequest = httpRequestParser.parse(request: requestData)
+                expect(parsedRequest).to(equal(expectedRequest))
```

Comment:
This assertion will (probably) not result in good error messages. My understanding is that you will not be given any information about what does not match on the request.

I would personally split this test into a single test per attribute you are parsing and not compare two requests like this.